docs/portal/src/common/Set/Settsx.md

```diff
@@ -0,0 +1,10 @@
+---
+sidebar_label: 'Set'
+sidebar_position: 1
+---
+# Set
+
+* js code for showing a group of cards
+* Set displays 4 elements by default, show more button will be there if there's more than 4 cards in a set
+    - if screen size is below 1200 px, it shows all elements at once
+* You can filter a set based on events that are currently happening, start time, and alphabetical using the info prop (TODO make sure this is true)
```

Comment:
we can, but i believe we just use one sorting method in the end. Bring up during meeting pls